cmd/agent/subcommands/run/command.go

```diff
@@ -430,8 +430,8 @@ func startAgent(
 		configService, err = remoteconfig.NewService()
 		if err != nil {
 			log.Errorf("Failed to initialize config management service: %s", err)
-		} else if err := configService.Start(context.Background()); err != nil {
-			log.Errorf("Failed to start config management service: %s", err)
```

Comment:
Is there a particular reason to remove this error log entirely ?
If possible having at least a warning would be nice

Reply:
Oh ok I didn't notice you changed the signature of the function too !